server/controller/processors/goalProcessor.ts

```diff
@@ -64,14 +70,21 @@ async function putGoalFeedback(req: Request, res: Response) {
         return;
     }
     res.sendStatus(StatusCode.OK);
+    const goal = await goalAPI.getGoalById(Number(req.params.id))
+    const goals = JSON.stringify(goal);
+    const module = await moduleAPI.getModuleById(Number(goals.replace("[{\"module_id\":", "").replace("}]", "")))
+    const modules = JSON.stringify(module);
+    const account = await loginAPI.getAccountById(Number(modules.replace("[{\"account_id\":", "").replace("}]", "")));
```

Comment:
If your goal is to simply get to the account_id, I might have a much simpler solution once I get the authentication pull request ready, but this works for now.